### PR TITLE
feat: product-page

### PR DESCRIPTION
**Pull request by vermasavita**
1. Created product page
2. Filter page with 4 different type of filter
3. For now, please avoid the header's style as that will be taken from style.css after merge. 

See page here: https://deploy-preview-4--bybook.netlify.app/pages/product/product.html

### CODE REVIEW

**Comment by AtulPant2704** [comment]
> _ No description provided. _

Add description for the PR and do add Codesandbox link or gif or screenshot, it makes reviewer's work easy.

**Comment by vermasavita** [comment]
@AtulPant2704 please check now

**Review by GauravDe-git — APPROVED** [review]
Looks good to go

**Comment by GauravDe-git** [comment]
> 1. Created product page
> 2. Filter page with 4 different type of filter
> 3. For now, please avoid the header's style as that will be taken from style.css after merge.
> 
> See page here: http://127.0.0.1:5501/pages/product/product.html

Just a suggestion, wanted to point out that this page link won't work for others because it is the local link created by the live server only for your pc. Instead, you can add a code sandbox link.

**Comment by ChhikaraBRUH** [comment]
> 1. Created product page
> 2. Filter page with 4 different type of filter
> 3. For now, please avoid the header's style as that will be taken from style.css after merge.
> 
> See page here: http://127.0.0.1:5501/pages/product/product.html

As you have hosted the repo on Netlify, you can share that link.
Here it is: https://deploy-preview-4--bybook.netlify.app/pages/product/product.html

File: pages/product/product.html [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+                <ul>
+                    <li>500</li>
+                    <li>1000</li>
+                    <li>5000</li>
+                </ul>
+                <label for="price-range-ipt"><input class="price-range-ipt" type="range" /></label>
+            </div>
+            <div class="product-category">
+                Category
+                <label for="category-checkbox"><input class="category-checkbox" type="checkbox" />Kindle Store
```
Comment (ChhikaraBRUH) [inline pages/product/product.html]: none of your labels for inputs are not working as intended.
Try changing "class" for inputs to "id". And I don't think they should be nested inside.
Read more about it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

**Comment by vermasavita** [comment]
Sure @ChhikaraBRUH 

Updated the link.

Please check and apporve

**Comment by vermasavita** [comment]
@GauravDe-git updated please check and approve

**Comment by ChhikaraBRUH** [comment]
@vermasavita Please update your input labels as well.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Example should be like:
`<input id="category-checkbox" type="checkbox" />`
`<label for="category-checkbox">Business & Economics</label>`

File: pages/product/product.html [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+
+        <div class="search-container">
+            <form action="">
+                <input id="search" type="search" placeholder="Search" />
+                <label for="search" class="bx bx-search-alt-2"></label>
+            </form>
+        </div>
+
+        <div class="action-icon">
+            <a href="#" class="login"><i class='bx bxs-user-circle'></i></a>
```
Comment (AtulPant2704) [inline pages/product/product.html]: You have not added any text over here for login link, it shows just a white area
navbar
.

**Comment by vermasavita** [comment]
@AtulPant2704  @ChhikaraBRUH please check now

**Review by ChhikaraBRUH — APPROVED** [review]
Good work

**Pull request merged**